en/competition/index.md

```diff
@@ -7,34 +7,34 @@ ref: competition
 image: "/banner/competition.jpg"
 ---
 
-After participating for many years to the Walking Machine SAE and the Eurobot subsequently, since 2016, we participate to the Robocup@Home international competition.
+After participating for many years in the Walking Machine SAE and the Eurobot competitions subsequently, since 2016, we are competing in the Robocup@Home international competition.
 
-## Robocup@Home in brief
+## Robocup@Home in a few lines
 
-The RoboCup@Home competition is held during the Robocup, the most important competition of Robotics in the world. The latter also presents the Soccer, Rescue and Junior categories. This event held once a year, brings together university and college from across the world.
+The RoboCup@Home is held during the Robocup, the most important robotics competition in the world. The latter also showcase the Soccer, Rescue, and Junior categories. This event, held once a year, brings together university and college from across the world.
```

Comment:
RoboCup-at-Home is held during Robocup, the most important robotics competition in the world. The latter also showcases the Soccer, Rescue, and Junior branches. This event, held once a year, brings together universities and colleges from around the world.